src/test/java/com/epam/ivankaliada/BasicSpringApp/service/EmployeeServiceImplTest.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.ivankaliada.BasicSpringApp.service;
+
+import com.epam.ivankaliada.BasicSpringApp.entity.Employee;
+import com.epam.ivankaliada.BasicSpringApp.repository.EmployeeRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.List;
+
+@ActiveProfiles("dev")
+@SpringBootTest
+public class EmployeeServiceImplTest {
+
+    @Autowired
+    private EmployeeService employeeService;
+    @Autowired
+    private EmployeeRepository employeeRepository;
+    @Autowired
+    private ApplicationContext applicationContext;
+
+    @Test
+    void shouldGetAllDbConfigurations() throws ParseException {
```

Comment:
There is a nice formatting approach for tests - **GIVEN-WHEN-THEN**. It makes tests more readable.
https://proandroiddev.com/given-when-then-our-testing-approach-c9087b291c36